### PR TITLE
Add drafts for Day 4, 6, and 8

### PR DESCRIPTION
**Pull request by excid3**
These could use a proof reading if anyone has time! 🙏

### CODE REVIEW

File: _drafts/acts_as_tenant.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
+# Day 8 - ActsAsTenant - Row-level multitenancy for Rails
+
+Written by [Chris Oliver](https://twitter.com/excid3) from [GoRails](https://gorails.com/) {% avatar excid3 %}
+
+## Multitenancy made easy with ActsAsTenant
```
Comment (swanson) [inline _drafts/acts_as_tenant.md:5]: Maybe add a sentence explaining what "multi-tenancy" is?

File: _drafts/pay.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
+# Day 6 - Pay - Payments for Ruby on Rails
+
+Written by [Chris Oliver](https://twitter.com/excid3) from [GoRails](https://gorails.com/) {% avatar excid3 %}
+
+## Accepting Payments in Rails
+
+Adding subscriptions and one-time purchases to your Rails app can be a bit tricky, especially with Strong Customer Authentication.
```
Comment (swanson) [inline _drafts/pay.md:7]: Link out to something about SCA?

**Review by swanson — APPROVED** [review]

**Comment by swanson** [comment]
Looks good to me, left two small comments.

**Comment by excid3** [comment]
Done and done. 👍

**Comment by swanson** [comment]
Thanks so much Chris!

**Pull request merged**